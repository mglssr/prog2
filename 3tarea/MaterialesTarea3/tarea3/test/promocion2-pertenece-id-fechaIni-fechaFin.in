crearPromocion 01/01/2002 01/06/2002 10 1
agregarAPromocion 5 0 1 2 3 4
imprimirPromocion
agregarAPromocion 5 5 6 7 8 9
imprimirPromocion
perteneceAPromocion 0
perteneceAPromocion 1
perteneceAPromocion 6
perteneceAPromocion 9
perteneceAPromocion 10
idPromocion
fechaInicioPromocion
fechaFinPromocion
liberarPromocion
Fin
