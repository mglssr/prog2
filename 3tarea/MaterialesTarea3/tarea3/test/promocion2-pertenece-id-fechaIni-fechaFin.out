Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> Promocion creada con exito.
2> Productos agregados a la promocion de forma exitosa.
3> Promocion #1 del 1/1/2002 al 1/6/2002
Productos: 0 1 2 3 4 
4> Productos agregados a la promocion de forma exitosa.
5> Promocion #1 del 1/1/2002 al 1/6/2002
Productos: 0 1 2 3 4 5 6 7 8 9 
6> El producto de id 0 pertenece a la promocion.
7> El producto de id 1 pertenece a la promocion.
8> El producto de id 6 pertenece a la promocion.
9> El producto de id 9 pertenece a la promocion.
10> El producto de id 10 NO pertenece a la promocion.
11> El id de la promocion es: 1.
12> La fecha inicio de la promocion es: 1/1/2002
13> La fecha fin de la promocion es: 1/6/2002
14> Promocion liberada con exito.
15> Fin.
